%-------------------------
% Resume in Latex
% Author : Sourabh Bajaj
% License : MIT
%------------------------

\documentclass[legalpaper,11pt]{article}

\usepackage{latexsym}
\usepackage[empty]{fullpage}
\usepackage{titlesec}
\usepackage{marvosym}
\usepackage[usenames,dvipsnames]{color}
\usepackage{verbatim}
\usepackage{enumitem}
%\usepackage[hidelinks]{hyperref}
\usepackage{hyperref}
\hypersetup{
    colorlinks=true,
    linkcolor=blue,
    filecolor=magenta,      
    urlcolor=blue%,
    %pdfpagemode=FullScreen
}

\usepackage{fancyhdr}
\usepackage[english]{babel}
\usepackage{tabularx}

\pagestyle{fancy}
\fancyhf{} % clear all header and footer fields
\fancyfoot{}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrulewidth}{0pt}

% Adjust margins
\addtolength{\oddsidemargin}{-0.5in}
\addtolength{\evensidemargin}{-0.5in}
\addtolength{\textwidth}{1in}
\addtolength{\topmargin}{-.5in}
\addtolength{\textheight}{1.0in}

\urlstyle{same}

\raggedbottom
\raggedright
\setlength{\tabcolsep}{0in}

% Sections formatting
\titleformat{\section}{
  \vspace{-4pt}\scshape\raggedright\large
}{}{0em}{}[\color{black}\titlerule \vspace{-5pt}]

%-------------------------
% Custom commands
\newcommand{\resumeItem}[2]{
  \item\small{
    \textbf{#1}{: #2 \vspace{-2pt}}
  }
}

\newcommand{\resumeSubheading}[4]{
  \vspace{-1pt}\item
    \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
      \textbf{#1} & #2 \\
      \textit{\small#3} & \textit{\small #4} \\
    \end{tabular*}\vspace{-5pt}
}

\newcommand{\resumeSubSubheading}[2]{
    \begin{tabular*}{0.97\textwidth}{l@{\extracolsep{\fill}}r}
      \textit{\small#1} & \textit{\small #2} \\
    \end{tabular*}\vspace{-5pt}
}

\newcommand{\resumeSubItem}[2]{\resumeItem{#1}{#2}\vspace{-4pt}}

\renewcommand{\labelitemii}{$\circ$}

\newcommand{\resumeSubHeadingListStart}{\begin{itemize}[leftmargin=*]}
\newcommand{\resumeSubHeadingListEnd}{\end{itemize}}
\newcommand{\resumeItemListStart}{\begin{itemize}}
\newcommand{\resumeItemListEnd}{\end{itemize}\vspace{-5pt}}

%-------------------------------------------
%%%%%%  CV STARTS HERE  %%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{document}

%----------HEADING-----------------
\begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}r}
  \textbf{{\Large Liezel Uy Tamon}} & Email : \href{mailto:liezel.tamon@imm.ox.ac.uk}{liezel.tamon@imm.ox.ac.uk} \& \href{mailto:liezeltamon@gmail.com}{liezeltamon@gmail.com}\\
  University of Oxford & Mobile : +44(0) 7789233360 \\
  \href{https://www.linkedin.com/in/liezel-tamon-613348174}{\textsc{LinkedIn}}
  \href{https://github.com/liezeltamon}{\textsc{GitHub}}
\end{tabular*}

%-----------EDUCATION-----------------
\section{EDUCATION}
  \resumeSubHeadingListStart
    \resumeSubheading
      {PhD in Medical Sciences (Computational Biology)}{Oxford, UK}
      {University of Oxford - \href{https://www.jardines.com/en/sustainability/our-strategy/shaping-social-inclusion/jardine-foundation?tab=scholarship-schemes}{Jardine Foundation Postgraduate Scholarship}}{Oct. 2018 -- Jun. 2023}
    \resumeSubheading
      {Master of Science in Molecular Biology}{Goettingen, DE}
      {\href{https://uni-goettingen.de/en/663690.html}{International Max Planck Research School, University of Goettingen} - IMPRS Scholarship}{Sep. 2016 -- Mar. 2018}
      \resumeSubheading
      {Bachelor of Science in Molecular Biology and Biotechnology}{Manila, PH}
      {University of the Philippines Diliman}{Jun. 2011 -- Jun. 2015}
      \resumeItemListStart
          \resumeItem{Winner, 2015 \href{https://www.sanger.ac.uk/about/study/the-sanger-prize/}{Sanger Institute Prize}}
          {International competition open to undergraduates from low- to upper-middle-income countries. The sole winner is granted a fully-funded, 3-month internship at the Wellcome Trust Sanger Institute in Cambridge, UK.}
      \resumeItemListEnd
  \resumeSubHeadingListEnd

% %-----------CURRENT ROLES-----------------
% \section{CURRENT ROLES}

% I manage simultaneous responsibilities, including teaching (in-person) computational biology courses on a termly basis (20 to 30 days per academic term) and working on purely computational projects as well as performing computational analyses of newly generated data in collaboration with experimental groups.

%   \resumeSubHeadingListStart

%     \resumeSubheading
%       {Postdoctoral computational biologist}{Oxford, UK}
%       {}{Jun. 2023 -- present}
%       \resumeItemListStart
%         \resumeItem{Supervised by Assoc. Prof. David Sims in MRC Weatherall Institute of Molecular Medicine - University of Oxford}{\href{https://www.rdm.ox.ac.uk/people/david-sims}{Transcriptional regulation in neuroscience}}
%         \resumeItem{Single-cell data analyses, collaboration}{Currently working on analysing single-cell transcriptomic datasets from two independent projects involving: 1) Genetically modified human brain organoids in collaboration with a group in Germany and 2) human-induced pluripotent stem cell (iPSC)-derived models for studying Parkinson's disease in collaboration with a group in Oxford}
%     \resumeItemListEnd
%     \vspace{0.4cm}
%     \resumeSubheading
%         {}{Oxford, UK}
%       {}{Jul. 2024 -- present}
%       \resumeItemListStart
%         \resumeItem{Project supervised by Assoc. Prof. Alejo J Nevado-Holgado in Department of Psychiatry - University of Oxford}{\href{https://www.psych.ox.ac.uk/team/alejo-nevado-holgado}{A.I. and bioinformatics applied to mental health}}
%         \resumeItem{Applying machine and deep learning algorithms to study dementia-related diseases}{Recognising the growing significance of ML/DL methods in medicine, I am volunteering to work with Alejo's research group to enhance my skills in applying these methods to various computational biology projects}
%      \resumeItemListEnd

%      \resumeSubheading
%       {Computational Biology instructor}{Oxford, UK}
%       {}{Jun. 2023 -- present}
%       \resumeItemListStart
%         \resumeItem{{\href{https://www.imm.ox.ac.uk/research/units-and-centres/mrc-wimm-centre-for-computational-biology/training/oxford-biomedical-data-science-training-programme}{Oxford Biomedical Data Science Training Programme}}}{Led by Assoc. Prof. David Sims; OBDS

%   \resumeSubHeadingListEnd
  
%-----------RESEARCH EXPERIENCE-----------------
\section{RESEARCH EXPERIENCE}
    \resumeSubHeadingListStart
        
        \resumeSubheading
        {Postdoctoral computational biologist \& data science trainer}{University of Oxford, UK}{Managing time between teaching (20 to 30 days per academic term) and research}{Jun. 2023 -- present}
            \resumeItemListStart
                \resumeItem{Single-cell research, collaboration \href{https://github.com/liezeltamon/utils/tree/main/R}{GitHub}}{Currently analysing single-cell RNA datasets from two independent projects: 1) Genetically modified human brain organoids with a group in Germany and 2) human-induced pluripotent stem cell (iPSC)-derived models for studying Parkinson's disease with a group in Oxford - Supervised by \href{https://www.rdm.ox.ac.uk/people/david-sims}{Assoc. Prof. David Sims - Transcriptional regulation in neuroscience}}
                \resumeItem{Teaching in \href{https://www.imm.ox.ac.uk/research/units-and-centres/mrc-wimm-centre-for-computational-biology/training/oxford-biomedical-data-science-training-programme}{Oxford Biomedical Data Science Training Programme}}{Genomics on the Linux command line, R for data science and genomics, Introduction to single-cell RNAseq data analysis using R. Contributed to course materials in {\href{https://github.com/kevinrue/OBDS_slides}{GitHub}}. Recently attended (Jan. 2024) the following modules: Python programming, Data science and Single-cell RNAseq using Python, to expand my skills.}
                 \resumeItem{Learning ML/DL algorithms to study dementia-related diseases}{ (Jul. 2024 -- present) Volunteering with the \href{https://www.psych.ox.ac.uk/team/alejo-nevado-holgado}{A.I. and bioinformatics applied to mental health} group in Oxford to develop my skills in applying these methods to various computational biology projects}
            \resumeItemListEnd
        \resumeSubheading
        {PhD student, computational biology}{University of Oxford, UK}{Transitioned from wet lab to pure computational research}{Oct. 2018 -- Jun. 2023}

            \resumeItemListStart
                \resumeItem{Thesis}{Analysis of long-range contacts across cell types outlines a core sequence determinant of 3D genome organisation \href{https://github.com/SahakyanLab/GenomicContactDynamics}{GitHub} \href{https://drive.google.com/file/d/1j3xg4ipr3ruG7vPtyXWO4wrDjJaRznfg/view?usp=share_link}{Manuscript} submitted - Supervised by \href{https://www.imm.ox.ac.uk/research/research-groups/sahakyan-group-integrative-computational-biology-and-machine-learning}{Dr. Aleksandr B. Sahakyan - Integrative Computational Biology and Machine Learning}}
                    \begin{itemize}
                        \resumeItem {Objective}{Leveraged multiple genomic contact datasets from various cell types and species, intersecting with other publicly available, processed datasets to delve into the core principles underlying 3D genome architecture, focusing on the contribution of sequence}
                        \resumeItem {Handled various datasets}{Genomic contacts from 3C-based methods, reference genomes, gene features and expression, somatic cancer SNV data, replication timing, DNA sequence features and annotations (e.g. k-meric composition, DNA secondary structure, repeat elements, motifs), $\approx$ 300 functional features from databases like ENCODE - primarily used custom code for analysing, combining, visualising and extracting features from these datasets}
                    \end{itemize}
            \resumeItemListEnd

            \resumeItemListStart
                \resumeItem{R \texttt{Shiny}}{\textbf{CoreGenomeExplorer} an interactive network visualisation of the core 3D genome \href{https://github.com/SahakyanLab/CoreGenomeExplorer/tree/master}{GitHub}}
                    \begin{itemize}
                        \item {Developed this application to facilitate interactive exploration and analysis of processed contact datasets}
                    \end{itemize}
            \resumeItemListEnd
            
            \resumeItemListStart
                \resumeItem{Tool development}{\textbf{ROptimus} a general purpose adaptive optimisation engine in R \href{https://cran.r-project.org/web/packages/ROptimus/index.html}{CRAN} \href{https://academic.oup.com/bioinformatics/article/39/5/btad292/7152277}{Paper}}
                    \begin{itemize}
                        \item {Contributed major algorithm modifications and a tutorial, in charge of journal revisions and CRAN deposition}
                        \item {Due to ROptimus' versatility in tackling various optimisation problems, I have used it for two different purposes, 1) generating control sets of contacts for my thesis and 2) developing a predictor for i-motif DNA stability (described below)}
                    \end{itemize}
            \resumeItemListEnd
            
            \resumeItemListStart
                \resumeItem{Modelling, machine learning}{Modelling stability indicators of i-motif secondary structures \href{https://github.com/SahakyanLab/iMotif_dev}{GitHub} \href{https://doi.org/10.1002/anie.202016801}{Paper}}
                    \begin{itemize}
                        \item {Used different methods to predict stability indicators (i.e. melting temperature and pH transition midpoint) of i-motif-forming DNA sequences: 1) \textsc{Eureqa} for symbolic regression, 2) \texttt{ROptimus} to adapt a G-quadruplex predictor for i-motifs, 3) \texttt{caret} \href{https://github.com/liezeltamon/ml-lib}{GitHub} and \texttt{xgboost} R packages to implement gradient boosting}
                        \item {Some experience with \href{https://github.com/autogluon/autogluon}{AutoGluon} for automated machine learning for my thesis}
                        \item {Attended in-person \href{https://training.cam.ac.uk/course/bioinfo-ml}{ML course} (Oct. 2-4, 
                        2019)}
                    \end{itemize}
            \resumeItemListEnd
            
            \resumeItemListStart
                \resumeItem{Consulting internship}{Provided a cost-benefit analysis of early diagnosis of dementia for \href{https://www.oxfordbraindiagnostics.com/}{Oxford Brain Diagnostics}, with patented technology for early and more accurate diagnosis}
                    \begin{itemize}
                        \item {\href{https://www.careers.ox.ac.uk/oxford-strategy-challenge}{The Oxford Strategy Challenge (TOSCA)} - The University of Oxford Careers Service offers a short consultancy training program where five-student teams work on projects for companies over 1.5 weeks}
                        \item {Provided scientific background on dementia and stage-specific interventions, among members that initiated to take a more quantitative approach estimating cost difference between standard and early dementia management per individual, highlighting benefits like reduced hospitalisations and cheaper total palliative care through early symptom management and lifestyle changes}
                    \end{itemize}
            \resumeItemListEnd
    
            \resumeItemListStart
                \resumeItem{Collaboration}{with \href{http://www.collepardolab.org/}{Computational Physics of Chromatin group} for thesis, with experimental chemistry groups abroad from Europe and China for the i-motif project, with group members for the ROptimus R package, with Oxford students from different fields for consulting internship}
            \resumeItemListEnd

\newpage

        \resumeSubheading
        {Other relevant skills}{Oct. 2018 -- present}{}{}
            \resumeItemListStart
                \resumeItem{Programming}{Proficient in R and Linux, familiar with Python (through online and in-person OBDS courses, \href{https://github.com/liezeltamon/advent-of-code}{self-learning}), familiar with \href{https://learn.365datascience.com/certificates/CC-1C72E62035/}{SQL} (through online course), IDEs (RStudio, VS Code, Jupyter Notebook)}
            \resumeItemListEnd

            \resumeItemListStart
                \resumeItem{Data visualisation}{R base, R external libraries (e.g. \texttt{ggplot2}, \texttt{Plotly}, \texttt{Shiny})}
            \resumeItemListEnd
            
            \resumeItemListStart
                \resumeItem{Handling and processing large-scale data}{Routinely using High-Performance Computing Linux-based clusters, developing parameterised workflows with modularised functions for parallelisation and scability}
                    \begin{itemize}
                        \item {Acquired skill during my PhD, where I mainly used a dataset of over 100 million genomic contacts from multiple cell types}
                    \end{itemize}
                \resumeItem{Documentation and reproducibility}{{\href{https://github.com/liezeltamon}{Git/GitHub}}, Git repository \href{https://github.com/liezeltamon/project-template}{project template}, Conda and \href{https://rstudio.github.io/renv/articles/renv.html}{\texttt{renv}} for reproducible environments, \href{https://quarto.org}{Quarto} and R Markdown, Overleaf online LaTeX editor, Microsoft Office}
                
            \resumeItemListEnd

        \resumeSubheading
        {Wet-lab experience}{Apr. 2014 -- Mar. 2018}{During Master's and Bachelor's studies}{DE and PH}
            \resumeItemListStart
                \resumeItem{Research topics}{Histone modifications, microRNA regulation, other epigenetic mechanisms in cancer, bone formation, muscular dystrophy}
                \resumeItem{Research methods}{Molecular biology experimental techniques (e.g. cloning, cell culture), worked with flies, \href{https://www.nature.com/articles/s41588-018-0244-3#data-availability}{ChRO-seq}, bulk RNA-seq and ChIP-seq data generation and analysis using \href{https://usegalaxy.org/}{\textsc{Galaxy}}, sequencing library preparation}
            \resumeItemListEnd
        
  \resumeSubHeadingListEnd
  
%-----------PUBLICATIONS-----------------
\section{PUBLICATION - \href{https://scholar.google.com/citations?user=XYWobTUAAAAJ&hl=en}{Google Scholar}}
    \begin{itemize}
        \item {Johnson NAG*, \textbf{Tamon L}*, Liu X, Sahakyan AB \texttt{ROptimus} \href{https://doi.org/10.1093/bioinformatics/btad292}{Bioinformatics 2023} \href{https://github.com/SahakyanLab/ROptimus}{GitHub}: Major algorithm modifications, CRAN deposition, publication}

            \item {Hamdan FH, Abdelrahman AM, Kutschat AP, Wang X, Ekstrom TL, Jalan-Sakrikar N, Wegner Wippel C, Taheri N, \textbf{Tamon L}...Najafova Z, Hessmann E, Truty MJ, Johnsen SA \href{http://dx.doi.org/10.1136/gutjnl-2022-328154}{BMJ Journals Gut 2023}: Contributed to generation of ChRO-seq data}

            \item {Cheng M, Qiu D, \textbf{Tamon L}...Sahakyan AB, Zhou J, Mergny JL \href{https://doi.org/10.1002/anie.202016801}{Angew Chem Int Ed Engl. 2021}: Contributed the i-motif DNA stability predictors}

            \item {Najafova Z, Liu P, Wegwitz F, Ahmad M, \textbf{Tamon L}...Johnsen SA, Tuckermann J \href{https://doi.org/10.1038/s41418-020-00614-w}{Nature Cell Death Differ. 2021}: Assisted with \textit{in vitro} experiments as part of my MSc}
            
            \item {Lemus-Diaz N, \textbf{Tamon L}, Gruber J. \href{https://doi.org/10.21769/BioProtoc.3000}{Bio-Protocol 2018}: Assisted with \textit{in vitro} experiments as part of my MSc rotation}
    \end{itemize}
          
\section{ADDITIONAL INFORMATION}
  \resumeSubHeadingListStart
    \resumeSubItem{Right to work in the UK}{I am a Filipino citizen with right to full-time work under a \href{https://www.gov.uk/graduate-visa}{Graduate Visa} valid until 31 July 2026}
    \resumeSubItem{Interests}
    {I love playing volleyball! I regularly play socially and also train as a member of the \href{http://www.oxfordvolleyball.co.uk}{Oxford Volleyball Club}. I also like playing racquet sports like tennis, squash and table tennis.}
  \resumeSubHeadingListEnd

\end{document}
